store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedViewsIT.java

```diff
@@ -180,39 +177,41 @@ public void shouldNotAddAndGetEdgeWithEntityGraph() throws OperationException {
         addBasicEdge();
 
         assertThatExceptionOfType(Exception.class)
-                .isThrownBy(() -> graph.execute(new GetAllElements.Builder()
+                .isThrownBy(() -> graph.execute(getFederatedOperation(new GetAllElements.Builder()
                         .view(new View.Builder()
-                                .edge(BASIC_EDGE)
+                                .edge(GROUP_BASIC_EDGE)
                                 .build())
-                        .option(FederatedStoreConstants.KEY_OPERATION_OPTIONS_GRAPH_IDS, ACCUMULO_GRAPH_WITH_ENTITIES)
-                        .build(), user))
-                .withMessage("Operation chain is invalid. Validation errors: \n"
-                        + "View is not valid for graphIds:[AccumuloStoreContainingEntities]\n"
-                        + "(graphId: AccumuloStoreContainingEntities) View for operation uk.gov.gchq.gaffer.operation.impl.get.GetAllElements is not valid. \n"
-                        + "(graphId: AccumuloStoreContainingEntities) Edge group BasicEdge does not exist in the schema");
+                        .build())
+                        .graphIdsCSV(GRAPH_ID_ACCUMULO_WITH_ENTITIES), user))
+                .withMessage("Operation chain is invalid. Validation errors: \n" +
```

Comment:
Can we either remove the \n, or mark this class as @DisabledOnOs(OS.WINDOWS) as we have done with other tests.